frontend/src/pages/Help.tsx

```diff
@@ -0,0 +1,187 @@
+          recent rows change as you navigate across different weeks or as you
+          add new time entries.
+        </p>
+        <Row
+          topic={exampleIAP}
+          days={[new Date()]}
+          rowHours={[1]}
+          rowEntries={[exampleEntry]}
+          getRowSum={() => 1}
+          onHide={() => {}}
+          isFav={false}
+          onToggleFav={() => {}}
+          onCellUpdate={() => {}}
+        ></Row>
+        <h2 className="help-subtitle">Adding new rows</h2>
+        <p className="help-info">
+          In order to add a new row, it is necessary to first enter a valid
+          issue number and then select an activity, which will be linked to the
+          issue. After clicking on the plus button, a new row will be appended
+          to the bottom of the list of recent rows.
+        </p>
+        <QuickAdd addIssueActivity={() => {}}></QuickAdd>
+        <p className="help-info">
+          If you have accidentally hidden a row, you can always bring it back by
+          using this feature.
+        </p>
+        <h2 className="help-subtitle">Adding or updating time entries</h2>
+        <p className="help-info">
+          One may log time by simply clicking on the cell corresponding to the
+          desired date. If the cell is emptied, the underlying time entry will
+          be deleted. It's of course also possible to update existing values of
+          a cell. All changes made to a cell are saved only after clicking on
+          the <b>"Save Changes"</b> button.
+        </p>
+        <p className="help-info">
+          It is also possible to add a comment to a time entry by clicking on
+          the triangle present at the top right of a cell. The color of this
+          icon is grey when a comment has not been saved yet:
+        </p>
+        <div className="centered">
+          <Cell
+            topic={exampleIAP}
+            date={new Date()}
+            hours={1}
+            comments={""}
+            entryId={1}
+            onCellUpdate={() => {}}
+          />
+        </div>
+        <p className="help-info">
+          However, once the comment has been saved along with the time entry,
+          the triangle turns black:
+        </p>
+        <div className="centered">
+          <Cell
+            topic={exampleIAP}
+            date={new Date()}
+            hours={1}
+            comments={"Test comment"}
+            entryId={1}
+            onCellUpdate={() => {}}
+          />
+        </div>
+        <h2 className="help-subtitle">Time travelling</h2>
+        <p className="help-info">
+          Users can navigate across different weeks by using the left and right
+          arrow or using the week picker. The week picker is displayed after
+          clicking on the button containing a calendar icon and the current week
+          number. It's worth noting that users can also go back to previously
+          visited weeks by using the browsers back button.
```

Comment:
Maybe add a sentence about the URL changing: 
"The URL displayed in your browser will show which year and week you are currently looking at. That makes it possible to note down a link to a certain week in urdr, in case you know you want to revisit that week at a later point."